### PR TITLE
Bump Jaeger tracer to 0.19.0, OT API to 0.22.0

### PR DESCRIPTION
**Pull request by pavolloffay**

### CODE REVIEW

**Comment by pavolloffay** [comment]
@rafabene this is missing to merge.

File: src/main/java/com/redhat/developers/msa/api_gateway/TracingConfiguration.java [hunk 1]
```diff
@@ -51,7 +51,7 @@
 
     @Bean
     public Tracer tracer() {
-        String jaegerURL = System.getenv("JAEGER_SERVER_URL");
+        String jaegerURL = System.getenv("JAEGER_SERVER_HOSTNAME");
```
Comment (pavolloffay) [inline src/main/java/com/redhat/developers/msa/api_gateway/TracingConfiguration.java]: It didn't work correctly before. It was expecting wrong env var.

**Comment by rafabene** [comment]
It has conflicts

**Comment by pavolloffay** [comment]
This is super weird, First I couldn't see any conflicts. When I looked through GH to branch `spring-boot-only` commit was there but PR still open (not merged). I pulled localy that that branch and pushed again and PR was automatically closed. What a magic?

**Pull request closed without merging**